compilers/concrete-compiler/compiler/tests/check_tests/Conversion/FHEToTFHECrt/mul_eint_int.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: concretecompiler --optimize-tfhe=false --action=dump-tfhe %s --large-integer-crt-decomposition=2,3,5,7,11 --large-integer-circuit-bootstrap=2,9 --large-integer-packing-keyswitch=694,1024,4,9 --v0-parameter=2,10,693,4,9,7,2 2>&1| FileCheck %s
```

Comment:
you can remove `--optimize-tfhe=false` for all those tests

Reply:
Sure.